WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -87,10 +98,20 @@ private extension BlogDashboardService {
     func parse(_ entity: BlogDashboardRemoteEntity?, blog: Blog, dotComID: Int) -> [DashboardCardModel] {
         let personalizationService = BlogDashboardPersonalizationService(repository: repository, siteID: dotComID)
         var cards: [DashboardCardModel] = DashboardCard.allCases.compactMap { card in
-            guard personalizationService.isEnabled(card),
-                  card.shouldShow(for: blog, apiResponse: entity) else {
+            guard personalizationService.isEnabled(card) else {
                 return nil
             }
+
+            guard card.shouldShow(
```

Comment:
I split this `guard` into two just because the `card.shouldShow...` parameters are so many and it would have looked awkward both as a one liner or with `card.shouldShow...` on a newline.